libpod/kube.go

```diff
@@ -489,15 +484,23 @@ func containerToV1Container(ctx context.Context, c *Container) (v1.Container, []
 		kubeContainer.Command = nil
 	}
 
+	if c.WorkingDir() != "/" && !reflect.DeepEqual(imgData.Config.WorkingDir, c.WorkingDir()) {
```

Comment:
what type is imgData.Config.WorkingDir that requires a deep equal here? can't it be cast to a string, which I assume will be less expensive?